client/src/components/jobCard/jobCard.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  Button,
+} from '@mui/material';
+import { Link } from 'react-router-dom';
+import './jobCard.css';
+
+function JobCard() {
+  return (
+    <div className="content">
+      <div className="job-card">
+        <Link to="/job/jobid">
+          <h2 className="h2">
```

Comment:
I would prefer to use typography mui component, Even If we want to keep use `h2` try to change `className`